Assets/Scripts/EndGoal.cs

```diff
@@ -8,28 +8,34 @@ public class EndGoal : MonoBehaviour
 {
 
     private string frogsRemainingText;
-    private int frogGoal = 1;
-
-
-    void Update()
+    private int frogGoal = 0;
+    private int frogAmount = 0; //amount of frogs in level
+    public int FrogGoal 
```

Comment:
It looks like FrogGoal is the amount of frogs which need to touch the goal in order to win and is not the same thing as the goal which the frogs are trying to reach (EndGoal). However, this is not super clear from the name. I am not sure what other name you could use, but I would recommend putting a summary whether you decide to change the name or not.